README.rst

```diff
@@ -4,6 +4,15 @@ pytoport
 A fairly normal way to generate FreeBSD port Makefiles straight from
 PyPI.
 
+Requirements
+------------
+
+In order to use pytoport, you should install the required dependencies:
+
+::
+
+    $ pip install -r requirements.txt
```

Comment:
The goal is to create a port so the requirement should be in ports, recommending using pip is then really weird and leads to installed untracked (by pkg) files on the system